jekyll/_cci2/language-ruby.md

```diff
@@ -7,60 +7,58 @@ categories: [language-guides]
 order: 8
 ---
 
-*[Tutorials & 2.0 Demo Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Ruby*
+*[Tutorials & 2.0 Sample Apps]({{ site.baseurl }}/2.0/tutorials/) > Language Guide: Ruby*
 
 This guide will help you get started with a Ruby on Rails application on CircleCI. 
 
 ## Overview
 
 If you’re in a rush, just copy the sample configuration below into a `.circleci/config.yml` in your project’s root directory and start building.
 
-Otherwise, we recommend reading our [walkthrough](#config-walkthrough) for a detailed explanation of our configuration.
```

Comment:
Why remove this?

Reply:
This is a link that jumps down the article from the third sentence, so it is better to just give some useful information here rather than a jump link.